src/Controller/AboutController.php

```diff
@@ -152,17 +152,27 @@ public function earlyCareerAction(Request $request) : Response
             'The community behind eLife wants to help address some of the pressures on early-career scientists');
 
         $arguments['body'] = [
-            new Paragraph('The community behind eLife – including the research funders who support the journal, the editors and referees who run the peer-review process, and our <a href="'.$this->get('router')->generate('about-people', ['type' => 'early-career']).'">Early-Career Advisory Group</a> – are keenly aware of the pressures faced by junior investigators, and are working to create a more positive publishing experience that will, among other things, help early-career researchers receive the recognition they deserve.'),
+            new Paragraph('The community behind eLife – including the research funders who support the journal, the editors and referees who run the peer-review process, and our Early-Career Advisory Group – are keenly aware of the pressures faced by early-stage investigators. That’s one reason we’re working to create a more positive publishing experience that will, among other things, help early-career researchers receive the recognition they deserve.'),
             new Paragraph('eLife supports and showcases early-career scientists and their work in a number of ways:'),
             new DefinitionList([
-                '<a href="'.$this->get('router')->generate('magazine').'">Magazine features</a>' => 'Early-career researchers and issues of concern to them are regularly featured in interviews, podcasts and articles in the Magazine section of eLife',
-                '<a href="'.$this->get('router')->generate('events').'">#ECRWednesday Webinars</a>' => 'A platform for the early-career community to share opportunities and explore issues around building a successful research career, on the last Wednesday of every month',
-                '<a href="'.$this->get('router')->generate('events').'">Events</a>' => 'We offer speakers and funding for scientific meetings organised for and by early-career researchers',
-                '<a href="'.$this->get('router')->generate('community').'">Travel grants</a>' => 'eLife offers funding to help early-career scientists get exposure and recognition for their work among leading scientists in their fields',
+                'Early Career Advisory Group' => 'eLife has invited a group of talented graduate students, postdocs and junior group leaders from across the world to our <a href="'.$this->get('router')->generate('about-people', ['type' => 'early-career']).'">Early-Career Advisory Group</a>. The ECAG acts as a voice for early-career researchers (ECRs) within eLife, representing their needs and aspirations and helping to develop new initiatives and shape current practices to change academic publishing for the better.',
+            ]),
+            new Paragraph('The role of the ECAG includes:'),
```

Comment:
I've just realised this is meant to be part of the `DefinitionList`.  Let me look at that.